From bd38ea69ed9fcb231ae0cd7693591be983cef84c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 07 Aug 2018 11:19:56 +0300
Subject: [PATCH] board: habana: improve the local_remote_common script

print_msg() function, precedes the message with asterisks. This should
not be done for the print_dmesg() function.
So do not use the print_msg() from inside the print_dmesg() function.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/local_remote_common b/board/habanalabs/common/local_remote_common
index 903c178..4b76aaa 100755
--- a/board/habanalabs/common/local_remote_common
+++ b/board/habanalabs/common/local_remote_common
@@ -8,8 +8,8 @@
 }
 
 function print_dmesg() {
-        print_msg "*************** dmesg start ***********************"
-        sudo dmesg -c
-        print_msg "*************** dmesg end *************************"
+	echo -e   "**************** dmesg start ***************************"
+	sudo dmesg -c
+	echo -e   "**************** dmesg end *****************************"
 }
 
